### PR TITLE
docs: update tekton and topology documentation with pod log permissions

### LINKED ISSUE #667: Tekton and Topology: Prerequisites in documentation insufficient to view logs
## What needs to be done
The current documentation for the Tekton and Topology plugin requires the user to create a Service Account for the Kubernetes plugin. It references the [Kubernetes RBAC section](https://backstage.io/docs/features/kubernetes/configuration/#role-based-access-control) in the backstage documentation, which does not mention the permissions required to display the pod logs.
image

To remedy this, the user would also need to provide the following permission (specifically the `pods/log` permission) to the Service Account's `ClusterRole`:
```yaml
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: view-pod-logs
rules:
  # Base for Kubernetes plugin
  - apiGroups:
      - ""
    resources:
      - pods
      - pods/log
     verbs:
      - get
      - list
```

### PR DESCRIPTION
**Pull request by Zaperex**
Fixes #667

### CODE REVIEW

File: plugins/tekton/README.md [hunk 1]
```diff
@@ -34,6 +36,14 @@ The Tekton plugin enables you to visualize the `PipelineRun` resources available
     metadata:
       name: backstage-read-only
     rules:
+      - apiGroups:
+          - ""
+        resources:
+          - pods/log
```
Comment (invincibleJai) [inline plugins/tekton/README.md:42]: is this needed, I don't think there is a resource `pod/log`.

cc @debsmita1

Comment (Zaperex) [reply plugins/tekton/README.md:42]: I wasn't able to get the logs to load without adding `pods/log` into the `ClusterRole`

Comment (invincibleJai) [reply plugins/tekton/README.md:42]: ah yeah my bad, this would be needed. 👍

**Comment by invincibleJai** [comment]
@Zaperex Thanks for the PR, The Semantic job is failing it should be `docs: *` i think

**Comment by debsmita1** [comment]
cc @hmanwani-rh

**Pull request merged**